logging/logger.go

```diff
@@ -21,6 +21,14 @@ type Logger interface {
 	Logf(classification Classification, format string, v ...interface{})
 }
 
+// LoggerFunc is a wrapper around a function to satisfy the Logger interface.
+type LoggerFunc func(classification Classification, format string, v ...interface{})
```

Comment:
what does `format` represent here? Slightly confusing naming. Should it be msg or something?

Reply:
`format` as in the format string. Given that this is has the `f` suffix similar to `fmt.Printf` etc this is the format string that the values will be templated into.